Animation.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.IO;
+using System.Collections.Generic;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using SixLabors.ImageSharp;
+using Rampastring.Tools;
+using SixLabors.ImageSharp.Formats;
+
+namespace Rampastring.XNAUI;
+
+class Frame
```

Comment:
I see `Frame` as a potential source of name clashes. Rename to `AnimationFrame`.

Reply:
Done by 67af83b4dd18808256ed10c1257b2560e28bfe94